subsystem/cluster_test.go

```diff
@@ -634,8 +634,9 @@ var _ = Describe("cluster update - BaseDNS", func() {
 
 var _ = Describe("cluster install", func() {
 	var (
-		ctx           = context.Background()
-		cluster       *models.Cluster
+		ctx     = context.Background()
+		cluster *models.Cluster
+		//nolint
```

Comment:
Why is it here for?

Reply:
Lint started complaining about overriding this value.  I did not touch this part of code at all.  So this tells lint not to complain.

Reply:
I think it should be resolved instead of disabling the linter.

Reply:
OK